packages/dapp/src/hooks/useCyberConnectWallet.ts

```diff
@@ -0,0 +1,28 @@
+import { CyberApp } from '@cyberlab/cyber-app-sdk';
+
+export const useCyberConnectWallet = () => {
+  const checkCyberConnectEnvironment = async () => {
+    // in Multisig env, window.parent is not equal to window
+    const isIframeEnvironment = window.parent !== window;
+
+    if (!isIframeEnvironment) {
+      return false;
+    }
+
+    const sdk = new CyberApp({
+      name: 'Li.Fi',
+      icon: 'https://github.com/lifinance/types/blob/main/src/assets/icons/bridges/lifuel.png',
+    });
```

Comment:
The name should be LI.FI. Can we also not use hard-coded icon links because we will move icons from there soon and they will be broken 🙂